src/encoded/docs/schema-changes.md

```diff
@@ -211,11 +271,40 @@ There are 2 categories when we talk about updating an existing schema:
 
 
         def test_{metadata_object}_lowercase_property_1(app, {metadata_object}_2):
-            migrator = app.registry['migrator']
-            value = migrator.upgrade('{metadata_object}', {metadata_object}_2, target_version='3')
+            value = upgrader.upgrade('{metadata_object}', {metadata_object}_2, target_version='3')
             assert value['schema_version'] == '3'
             assert value['property_1'] == 'value 1'
-            
+
+**Specific example from the library object upgrade**
+
+        import pytest
+
+
+        @pytest.fixture
+        def library(lab, award):
+            return {
+                'award': award['uuid'],
+                'lab': lab['uuid'],
+                'nucleic_acid_term_id': 'SO:0000352',
+                'nucleic_acid_term_name': 'DNA',
+            }
+
+        @pytest.fixture
+        def library_8(library_3):
+            item = library_3.copy()
+            item.update({
+            'schema_version': '8',
+            'status': "in progress"
+        })
+        return item
+
+        def test_upgrade_library_8_to_9(upgrader, library_8):
+            value = upgrader.upgrade('library', library_8, target_version='9')
+            assert value['schema_version'] == '9'
+            assert isinstance(value['fragmentation_methods'], list)
+            assert value['fragmentation_methods'] == ['shearing (Covaris generic)']
+            assert 'fragmentation_method' not in value
+
```

Comment:
You can be less detailed here, you don't need to have proper code for the fixture and the test, just include the relevant lines that you would like the reader to concentrate on